float64[] theta_ref
float64[] theta_min
float64[] theta_max
geometry_msgs/Pose pose
---
ik[] ik_solution
bool status

